pkg/cmd/release/view/view.go

```diff
@@ -217,3 +310,104 @@ func floatToString(f float64, p uint8) string {
 	idx := strings.IndexRune(fs, '.')
 	return fs[:idx+int(p)+1]
 }
+
+func matchTag(pattern, tag string) (bool, error) {
+	g, err := glob.Compile(pattern)
```

Comment:
Semver constraints like `1.2.*` are not globs: they shouldn't support expressions like `**`, `?`, `[...]`, nor `{...}`. Finally, a pattern like `*.*.2` should not be valid input. Therefore, I would not suggest using the glob library here.

Instead, I would just do inline string replacement to convert any expression like `X.Y.*` to a semver range `X.Y.0` – `X.(Y+1).0` (non-inclusive).

Reply:
Sure, looks like a correct approach.